webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocumentUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.box.l10n.mojito.android.strings;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerFactory;
+
+public abstract class AndroidStringDocumentUtils {
```

Comment:
why abstract class? doesn't seem to be extended anywhere. if you want to prevent instantiation, just a private constructor will do + final class?

Reply:
Updated